### PR TITLE
rename list-group

### PR DESCRIPTION
**Pull request by kulotsystems**

### CODE REVIEW

File: assets/css/pages.css [hunk 1]
```diff
@@ -47160,29 +47160,29 @@ tr.show-desktop {
   outline: 0 !important;
 }
 @media (min-width: 744px) {
-  ._dkkqb8 {
+  .btn-outline-hover {
```
Comment (gbertl) [inline assets/css/pages.css:47163]: please rename this to something else we already have btn-outline. i think panel or card

Comment (kulotsystems) [reply assets/css/pages.css:47163]: Alright... I'm thinking of list-group for this.

Comment (gbertl) [reply assets/css/pages.css:47163]: can you deploy it to github pages po para mareview ko live nalang kasi local ginagawa ko

Comment (kulotsystems) [reply assets/css/pages.css:47163]: okay sige po

Comment (kulotsystems) [reply assets/css/pages.css:47163]: https://kulotsystems.github.io/crispy-train/

Comment (gbertl) [reply assets/css/pages.css:47163]: ok po sa list-group for now

File: become-a-host/privacy-type.html [hunk 2]
```diff
@@ -167,10 +167,10 @@ <h1 class="_hvpbea" tabindex="-1">
                     <div class="_tdjw9nc" style="transition: opacity 0.6s">
                       <div class="_10mqyec">
                         <span style="font-size: 0"></span>
-                        <div class="_15gowa3" role="radiogroup" id="FMP-target">
-                          <div class="_7ievarx" style="animation-delay: 400ms">
+                        <div class="list-group" role="radiogroup" id="FMP-target">
+                          <div class="list-group-item" style="animation-delay: 400ms">
                             <button
-                              class="_28hxjy8"
+                              class="list-group-item-content--active"
```
Comment (gbertl) [inline become-a-host/privacy-type.html:173]:
```suggestion
                              class="list-group__btn--active"

```

File: become-a-host/privacy-type.html [hunk 3]
```diff
@@ -167,10 +167,10 @@ <h1 class="_hvpbea" tabindex="-1">
                     <div class="_tdjw9nc" style="transition: opacity 0.6s">
                       <div class="_10mqyec">
                         <span style="font-size: 0"></span>
-                        <div class="_15gowa3" role="radiogroup" id="FMP-target">
-                          <div class="_7ievarx" style="animation-delay: 400ms">
+                        <div class="list-group" role="radiogroup" id="FMP-target">
+                          <div class="list-group-item" style="animation-delay: 400ms">
```
Comment (gbertl) [inline become-a-host/privacy-type.html:171]:
```suggestion
                          <div class="list-group__item" style="animation-delay: 400ms">

```

File: become-a-host/privacy-type.html [hunk 4]
```diff
@@ -182,11 +182,11 @@ <h1 class="_hvpbea" tabindex="-1">
                             </button>
                           </div>
                           <div
-                            class="_7ievarx"
+                            class="list-group-item"
                             style="animation-delay: 449.7412007086385ms"
                           >
                             <button
-                              class="_dkkqb8"
+                              class="list-group-item-content"
```
Comment (gbertl) [inline become-a-host/privacy-type.html:189]:
```suggestion
                              class="list-group__btn"

```

**Review by gbertl — CHANGES_REQUESTED** [review]

**Comment by gbertl** [comment]
https://scalablecss.com/bem-nesting-grandchild-elements/

File: become-a-host/privacy-type.html (hunk 4 above)
Comment (kulotsystems) [reply become-a-host/privacy-type.html:189]: Alright, got it..

**Comment by gbertl** [comment]
Introduce .list-group component. -> rename list-group. Also add sa google sheet yung old and new classnames

**Comment by gbertl** [comment]
Please squash your  last 3 commits into one. tapos pafollow nalang po ng gantong format "rename list-group" yung parent block lang yung nasa commit, ginagamit ko kasi sya pag nag eextract ako

**Pull request merged**